lib/services/OrderService.js

```diff
@@ -7,9 +7,25 @@ module.exports = class OrderService {
       process.env.ORDER_HANDLER_NUMBER,
       `New Order received for ${value.quantity}`
     );
-
     const order = await Order.insert(value);
+    return order;
+  }
+
+  static async updateOrder(value) {
+    await sendSms(
+      process.env.ORDER_HANDLER_NUMBER,
+      `Order ${value.id}, updated quantity: ${value.quantity}`
+    );
+    const order = await Order.update(value);
```

Comment:
This references a method that doesn't exist on `Order` - if you use `updateById` here it should do what you expect.